src/PropValidations.tsx

```diff
@@ -1,52 +1,46 @@
-import * as React from 'react';
-
-import includes = require('lodash/includes');
-
 import { safeStringify } from '@stoplight/json';
+import * as React from 'react';
 
+import { IProp, IResolvedProp } from './types';
 import { isCombiner } from './util/isCombiner';
 import { pickValidations } from './util/pickValidations';
 
-export const PropValidations = ({ prop }: { prop: any }) => {
+export const PropValidations: React.FunctionComponent<{ prop: IProp | IResolvedProp }> = ({ prop }) => {
   if (!isCombiner(prop)) {
     const validations = pickValidations(prop);
 
-    const elems = [];
-    for (const k in validations) {
-      if (!Object.prototype.hasOwnProperty.call(validations, k)) {
-        continue;
-      }
-
-      let v = validations[k];
-
-      let type = typeof v;
-
-      if (k === 'default' && includes(['object', 'boolean'], type)) {
-        v = safeStringify(v);
-
-        type = typeof v;
-      }
-
-      if (type === 'object') {
-        continue;
-      } else if (type === 'boolean') {
-        elems.push(
-          <div key={k}>
-            <b>{k}:</b> {v.toString()}
-          </div>
-        );
-      } else {
-        elems.push(
-          <div key={k}>
-            <b>{k}:</b> {v}
-          </div>
-        );
-      }
-    }
-
-    if (elems.length) {
-      return elems;
-    }
+    return (
+      <>
+        {Object.entries(validations).reduce(
```

Comment:
I'd personally use `.map` + `.filter` just because it's less error prone. Actually you don't even need the filter because you can just return `null` for those elements and React won't render anything in those spots.